data/settings_mapping.json

```diff
@@ -566,6 +567,7 @@
             "sfx_menu_cursor",
             "sfx_menu_select",
             "sfx_low_hp",
+            "slowdown_music_when_lowhp",
```

Comment:
I would move this into the "General" section alongside the other setting for 2 reasons:

* It just makes more sense to me for it to be there alongside the other music-altering settings
* It isn't (and shouldn't be) randomized by the randomize option, but is still grayed out when the randomize option is selected. This could of course be solved by randomizing the individual settings in this section instead of the whole section (like we do for voices) but just moving it to the general section is cleaner.